### ISSUE TITLE
Also take a look at (move to) Prismatic's work

### ISSUE THREAD

**Issue by zeeshanlakhani** [zeeshanlakhani/schema-gen#18]
https://github.com/Prismatic/schema/blob/121b789db4e9bc658adb378f1ff05f3bb0f103aa/README.md#generation-and-completion

@gfredericks @sgrove

**Comment by gfredericks** [comment]
I'll probably be trying that stuff out at work soon

**Comment by zeeshanlakhani** [comment]
@gfredericks make any progress w/ the Prismatic stuff?

**Comment by gfredericks** [comment]
Yeah I think it's customizable enough to be a solid successor to schema-gen. Thanks for keeping this going in the meantime.
